symphony/bdk/core/service/message/messageml_util.py

```diff
@@ -0,0 +1,51 @@
+"""This module handles pre-processing for an outgoing message in order to have a valid messageML format.
+"""
+import re
+
+
+def escape_special_chars(raw_text: str):
+    """ Replace all special characters placed within the messageML that must be HTML-escaped
+    to have a valid MessageML format.
+
+    :param raw_text: text to be parsed
+    :return: text in a valid messageML format
+    """
+    current_parsed_index = 0
+    parsed_text = ""
+    matches = re.finditer(_pattern(), raw_text)
+    for match in matches:
+        parsed_text += raw_text[current_parsed_index:match.start()] + _replacement(match.group(0))
+        current_parsed_index = match.end()
+    return parsed_text + raw_text[current_parsed_index:]
+
+
+def _pattern():
+    return "|".join(_special_chars_dic)
```

Comment:
This is me being very picky: maybe a t for dicT at the end. _special_chars_dict instead of _special_chars_dic

Reply:
I thought it was already called like that! XD